### PR TITLE
Checks live seekable range and HTMLMediaElement.seekable

### PR DESCRIPTION
**Pull request by tidoust**
These tests check the setLiveSeekableRange and clearLiveSeekableRange methods, and their impact on the seekable attribute of the media element.

Note that the steps of the HTMLMediaElement.seekable attribute algorithm that do not depend on the live seekable range are not tested here.

### CODE REVIEW

File: media-source/mediasource-liveseekable.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html>
```
Comment (wolenetz) [inline media-source/mediasource-liveseekable.html]: style nit: no <html> <head> or <body> is what I was recently told is the new latest and greatest wpt style. Also, left-margin of 0 for the outermost layer of script body.

File: media-source/mediasource-liveseekable.html [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    <head>
+        <title>Checks setting/clearing the live seekable range and HTMLMediaElement.seekable</title>
+        <script src="/resources/testharness.js"></script>
+        <script src="/resources/testharnessreport.js"></script>
+        <script src="mediasource-util.js"></script>
+        <link rel="stylesheet" href="/resources/testharness.css">
```
Comment (wolenetz) [inline media-source/mediasource-liveseekable.html]: nit: this css inclusion is now automatic. drop this redundant line please

File: media-source/mediasource-liveseekable.html [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+    </head>
+    <body>
+        <div id="log"></div>
```
Comment (wolenetz) [inline media-source/mediasource-liveseekable.html]: nit: drop this unnecessary legacy log div line please.

**Comment by wolenetz** [comment]
looks good to me % really minor nits. As a bonus, my in-progress Chromium CL passes this test as-is. I'll reference my Chromium CL shortly once it's been uploaded for review there.

**Comment by wolenetz** [comment]
FYI the Chromium implementation of {set,clear}LiveSeekableRange is currently in codereview at https://codereview.chromium.org/2105593002/.

**Comment by tidoust** [comment]
Thanks for the review @wolenetz. Indeed, I see that the wpt documentation provides [test templates](http://testthewebforward.org/docs/test-templates.html) which I had overlooked. I'll go through the other pending PR and update those that create new test files as well (I won't touch on existing files for now to ease review, but we could perhaps update them in a final pass as well).

Merging this PR now.

**Pull request merged**